### PR TITLE
Add canary release workflow

### PR DESCRIPTION
**Pull request by a26620236**
# Purpose

Asana Task: https://app.asana.com/0/0/1202114099601734/f
原本 Travis CD 壞掉了，這邊改用 github action 去做 canary release 的自動化

# Changes

- a list of what have been done
- maybe some code change

# Risk

Usually none, if you have any please write it here.

# TODOs

- [ ] Describe what should be done outside of this PR
- [ ] Maybe in other PRs or some manual actions.

### CODE REVIEW

File: .github/workflows/canary_release.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#      - develop
+
+jobs:
+  canary-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
```
Comment (chenesan) [inline .github/workflows/canary_release.yml:21]: 有點忘了，現在 gypcrete 還是用 node 12 嗎

**Review by chenesan — APPROVED** [review]
讚讚

File: .github/workflows/canary_release.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      - reopened
+#  push:
+#    branches:
+#      - develop
+
+jobs:
+  canary-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (kyoyadmoon) [inline .github/workflows/canary_release.yml:18]: 這裡加上這個看看
```suggestion
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
```

File: .github/workflows/canary_release.yml (hunk 1 above)
Comment (a26620236) [reply .github/workflows/canary_release.yml:23]: 對的，印象中這邊用最新的 16 會噴錯

Comment (a26620236) [reply .github/workflows/canary_release.yml:23]: @chenesan 
image

Comment (chenesan) [reply .github/workflows/canary_release.yml:23]: 原來 gypcrete 本人還沒升到 16 xd

File: .github/workflows/canary_release.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+on:
+# todo: remove this after complete
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+#  push:
+#    branches:
+#      - develop
```
Comment (kyoyadmoon) [inline .github/workflows/canary_release.yml:12]: 這個要記得改回來 😂

Comment (a26620236) [reply .github/workflows/canary_release.yml]: 我少推一個 commit ...

**Review by kyoyadmoon — APPROVED** [review]
讚讚 👍🏼

**Pull request merged**